NuGet.config

```diff
@@ -11,6 +11,7 @@
     <add key="dotnet8" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" />
     <add key="dotnet8-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8-transport/nuget/v3/index.json" />
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-maintenance-packages-3b65679f" value="https://dnceng.pkgs.visualstudio.com/public/_packaging/darc-pub-dotnet-maintenance-packages-3b65679f/nuget/v3/index.json" />
```

Comment:
Is it OK for us to have packages with a dependency from a non-product feed?

Reply:
This was temporary. I am removing it now.